rudder-web/src/main/resources/load-page/rudder-not-loaded.html

```diff
@@ -28,6 +27,7 @@
           <li>A non-existing URL</li>
           <li>An essential component (LDAP or PostgreSQL) that is not accessible</li>
           <li>The web application being unable to load</li>
+          <li>rudder-init script has not been executed before the first run</li>
```

Comment:
Please use the text: `A non initialised Rudder installation. Please run /opt/rudder/bin/rudder-init to set it up.`